markdown/reference/sql/DROP-CONVERSION.html.md.erb

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
Something is wrong here - this content is a duplicate of `DROP-CAST.html.md.erb`